pages/404.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div id="center">
+    <h1>404</h1>
+    <canvas ref="canvas"></canvas>
+  </div>
+</template>
+
+<style scoped>
+#center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+}
+h1 {
+  padding-top: 40px;
+  font-weight: bold;
+  font-size: 60px;
+}
+canvas {
+  margin-top: 5vh;
+  height: 60vh;
+}
+</style>
+
+<script>
+export default {
+  mounted() {
```

Comment:
burdaki kodları bi method'a al ismi `setMeme` gibi bişey olabilir mounted içerisinde `this.setMeme()` olarak çağır